components/dialogs/b2b/AccountHierarchyEditFormDialog/AccountHierarchyEditFormDialog.tsx

```diff
@@ -0,0 +1,42 @@
+import { useTranslation } from 'next-i18next'
+
+import { AccountHierarchyEditForm } from '@/components/b2b'
+import { KiboDialog } from '@/components/common'
+import { UpdateCustomerB2bAccountParams } from '@/lib/types'
+
+import { B2BAccount } from '@/lib/gql/types'
+
+interface AccountHierarchyEditFormDialogProps {
+  accounts: B2BAccount[]
+  accountToEdit: B2BAccount
+  formTitle?: string
+  onSave: (data: UpdateCustomerB2bAccountParams) => void
+  onClose: () => void
+}
```

Comment:
instead of sending accountToEdit just send currentParent i.e. accountToEdit.companyOrOrganization

Reply:
Changed accountToEdit to "b2BAccount", which is more meaningful.
We cannot use currentParent as currentParent is not required to edit parent.